scripts/generate_docs.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2017-2020 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+set -ex
+
+my_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+root_path="$my_path/.."
+version=$(git describe --abbrev=0 --tags || echo "0.0.0")
+modules=( NIOSSH )
```

Comment:
I see there a re a f ew other modules in the repo, is including only NIOSSH in the docs intentional?

Reply:
Yes, the other modules are binaries.